plugins/inputs/postgresql/connect.go

```diff
@@ -79,21 +75,3 @@ func ParseURL(uri string) (string, error) {
 	sort.Strings(kvs) // Makes testing easier (not a performance concern)
 	return strings.Join(kvs, " "), nil
 }
-
-func Connect(address string) (*sql.DB, error) {
```

Comment:
Yup, I added the connect method originally. I'm just fixing the regression for the code I commited. You'll notice I changed postgresql_extensible as well in the diff.

Thanks for the review. =)